content/rooms/v10.md

```diff
@@ -58,12 +58,12 @@ Room version 10 is based upon room version 9 with the following considerations.
 The event format is unchanged by this room version. See [below](#event-format-1)
 for details on the current event format.
 
-#### Deprecated formatting
+#### Deprecated event content schemas
 
 While this room version does not change the event format specifically, some
 deprecated behaviours are strictly no longer supported.
 
-##### Power levels must be integers
+##### Values in `m.room.power_levels` events must be integers
 
 In other room versions, such as [v9](/rooms/v9/#power-levels-accepted-as-strings),
```

Comment:
```suggestion
In other room versions, such as [v9](/rooms/v9/#mroompower_levels-events-accept-values-as-strings),
```

Reply:
oops, I guess that's an important detail - thanks 😅